resources/assets/coffee/react/_components/comments-manager.coffee

```diff
@@ -81,8 +81,14 @@ export class CommentsManager extends React.PureComponent
 
 
   toggleShowDeleted: =>
+    nextState = !uiState.comments.isShowDeleted
+
     runInAction () ->
-      uiState.comments.isShowDeleted = !uiState.comments.isShowDeleted
+      uiState.comments.isShowDeleted = nextState
+
+    $.ajax laroute.route('account.options'),
+      method: 'PUT'
+      data: user_profile_customization: comments_show_deleted: nextState
```

Comment:
it's time to just move this to the store

Reply:
or maybe when splitting it off from `UIStateStore`. The entirety of ui store for comment is currently part of `UIStateStore`. Or more like everything is part of it.